src/main/java/com/icthh/xm/ms/entity/contrib/jsonpath/FixedDefaultJsonUnmarshaller.java

```diff
@@ -1,259 +0,0 @@
-package com.icthh.xm.ms.entity.contrib.jsonpath;
-
-import static com.fasterxml.jackson.databind.DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES;
-import static com.jayway.jsonpath.JsonPath.using;
-import static java.lang.String.format;
-
-import com.fasterxml.jackson.contrib.jsonpath.JsonUnmarshaller;
-import com.fasterxml.jackson.contrib.jsonpath.util.CandidateField;
-import com.fasterxml.jackson.contrib.jsonpath.util.JsonPathUtil;
-import com.fasterxml.jackson.contrib.jsonpath.util.ReflectionUtil;
-import com.fasterxml.jackson.databind.JavaType;
-import com.fasterxml.jackson.databind.ObjectMapper;
-import com.jayway.jsonpath.Configuration;
-import com.jayway.jsonpath.PathNotFoundException;
-import com.jayway.jsonpath.ReadContext;
-import com.jayway.jsonpath.spi.json.JacksonJsonProvider;
-import java.io.IOException;
-import java.lang.reflect.Field;
-import java.lang.reflect.ParameterizedType;
-import java.lang.reflect.Type;
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.List;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
-/**
- * The default implementation of {@link JsonUnmarshaller}.
- */
-public class FixedDefaultJsonUnmarshaller implements JsonUnmarshaller {
```

Comment:
Do we really need to delete this class? i do not remember if it was generated. Seems like its our class. 
Please recheck.

Reply:
This is very old class that was commited in 2017

`amedvedchuk 30.09.17, 15:35 Initial sources commit`

Class is not used anywhere. Library which this class extend also very old (last commit in 2015)
https://github.com/elasticpath/json-unmarshaller

So yes I think it can be removed